packages/forms/src/__tests__/form.test.tsx

```diff
@@ -249,13 +254,62 @@ describe('Form', () => {
 
     await waitFor(() => expect(mockFn).toHaveBeenCalledTimes(1))
     expect(mockFn).toBeCalledWith(
-      { userId: undefined, groupId: undefined },
+      { userId: null, groupId: null },
       expect.anything() // event that triggered the onSubmit call
     )
   })
 
+  it('ensures required textField is enforced by validation', async () => {
+    const mockFn = jest.fn()
+
+    render(
+      <Form onSubmit={mockFn}>
+        <TextField
+          name="userId2"
+          defaultValue=""
+          validation={{ required: true }}
+        />
+        <Submit>Save</Submit>
+      </Form>
+    )
+
+    await act(async () => {
```

Comment:
Can we get rid of the `act` here?

Reply:
Yes, that was my fault ... we had discussed that in February.  Thanks for the reminder!